pandas/io/parquet.py

```diff
@@ -277,6 +277,17 @@ def read_parquet(path, engine='auto', columns=None, **kwargs):
     Returns
     -------
     DataFrame
+        A parquet file is returned as two-dimensional data structure with
```

Comment:
(To the other maintainers) do we have a general policy on these? I don't think this description adds much beyond just having the return type.

Reply:
We do technically require a description for the Returns but I would be on favor of relaxing hat requirement. Agreed this doesn’t add value and may not in most instances

Reply:
```suggestion
        DataFrame with appropriate data.
```

As mentioned probably shouldn't need this at all, but can at least shorten to get CI to pass in the meantime